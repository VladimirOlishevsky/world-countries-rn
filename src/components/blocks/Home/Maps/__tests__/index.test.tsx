import React from "react";
import { Maps } from "..";
import renderer from 'react-test-renderer';

const mockStore = {
  countryStore: {
    latlng: [0, 1]
  }
}

jest.mock('store/index', () => ({
  getRootStore: () => mockStore,
}));


jest.mock('react-native-maps', () => {
  const { View } = require('react-native');
  const MockMapView = () => {
    return <View></View>;
  };
  const MockMarker = () => {
    return <View></View>;
  };
  return {
    __esModule: true,
    default: MockMapView,
    Marker: MockMarker,
  };
});

describe('Maps', () => {
  test('render', () => {
    const tree = renderer.create(<Maps />).toJSON();
    expect(tree).toMatchSnapshot();
  })
});