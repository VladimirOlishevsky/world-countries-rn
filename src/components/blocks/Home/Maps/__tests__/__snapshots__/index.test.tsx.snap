// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Maps render 1`] = `<View />`;
